libraries/AP_GPS/AP_GPS_NMEA.h

```diff
@@ -146,7 +147,7 @@ class AP_GPS_NMEA : public AP_GPS_Backend
     uint32_t _last_RMC_ms;
     uint32_t _last_GGA_ms;
     uint32_t _last_VTG_ms;
-    uint32_t _last_HDT_ms;
+    uint32_t _last_HDT_THS_ms{0};
```

Comment:
we don't need to init this to zero as we auto-init all allocated objects to zero. Adding the init consumes flash space unnecessarily